lambda/src/main/resources/db/migration/V46__create_file_property_v2.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE IF NOT EXISTS public."FilePropertyV2"
+(
+    "Name"              text          NOT NULL,
+    "Description"       text,
+    "Fullname"          text,
+    "CreatedDatetime"   timestamptz,
+    "ModifiedDatetime"  timestamptz,
+    "UserId"            uuid,
+    "PropertyType"      text,
+    "Datatype"          text,
+    "Editable"          boolean,
+    "MutliValue"        boolean,
+    "PropertyGroup"     text
+);
+
+ALTER TABLE public."FilePropertyV2"
+    ADD CONSTRAINT "FilePropertyV2_pkey"
+        PRIMARY KEY ("Name");
+
+INSERT INTO "FilePropertyV2" ("Name","Description","Fullname","CreatedDatetime","ModifiedDatetime","UserId","PropertyType","Datatype","Editable","MutliValue","PropertyGroup")
+VALUES
+    ('ClientSideOriginalFilepath','The original filepath','Filepath','2021-02-11 08:07:18.000 +0000',NULL,cast('eed88948-daf0-49c6-b511-c61dc6930433' as uuid),'System','text',false,false,NULL),
```

Comment:
The date's represent the `CreatedDatetime`. Should we not be generating these when we add the tables, instead of using an old date?